topics/computational-chemistry/tutorials/zauberkugel/tutorial.md

```diff
@@ -0,0 +1,311 @@
+ 
+## Ligand hydration
+
+More often than not, the bioactive form of a compound is its predominant form at physiological pH (7.4). In this step, we predict the most probable ionization state(s) of the query ligand at pH 7.4 with the cheminformatics toolkit OpenBabel ({% cite OBoyle2011 %}).
+
+> ### {% icon hands_on %} Hands-on 4: Add hydrogen atoms
+>
+> 1. {% tool [Add hydrogen atoms](toolshed.g2.bx.psu.edu/repos/bgruening/openbabel_addh/openbabel_addh/3.1.1+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Molecular input file"*: `output` (Input dataset)
+>    - *"Add hydrogens to polar atoms only (i.e. not to carbon atoms)"*: `Yes`
+>
+{: .hands_on}
+
+> ### {% icon question %} Question
+>
+> Nitrogen-containing functional groups are known to be basic. Which of them present in staurosporine do you expect to be protonated at pH 7.4, and which not? And why?
+>
+> > ### {% icon solution %} Solution
+> >
+> > Only the secondary N-methylamino group will be protonated because indoles much like aromatic amides are typically not basic.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Splitting ePharmaLib into individual pharmacophores
+
+The ePharmaLib subset representing human protein targets (*ePharmaLib_PHARAO_human.phar*) is a concatenated file containing 5,010 individual pharmacophore files. In order to speed up our analysis, it is preferable to split the dataset into individual files in order to perform several pharmacophore alignments in parallel, using Galaxy's collection functionality.
+
+> ### {% icon hands_on %} Hands-on 5: Splitting ePharmaLib
+>
+> 1. {% tool [Split file](toolshed.g2.bx.psu.edu/repos/bgruening/split_file_to_collection/split_file_to_collection/0.5.0) %} with the following parameters:
+>    - *"Select the file type to split"*: `Generic`
+>        - {% icon param-file %} *"File to split"*: `output` (Input dataset)
+>        - *"Method to split files"*: `Specify record separator as regular expression`
+>            - *"Regex to match record separator"*: `\$\$\$\$`
+>            - *"Split records before or after the separator?"*: `After`
+>        - *"Specify number of output files or number of records per file?"*: `Number of records per file ('chunk mode')`
+>        - *"Base name for new files in collection"*: `epharmalib`
+>        - *"Method to allocate records to new files"*: `Maintain record order`
+>
+>    > ### {% icon details %} What is a PHAR file?
+>    >
+>    > A PHAR file is essentially a series of lines containing the three-dimensional coordinates of pharmacophoric features and excluded spheres. The first column specifies a feature type (e.g. HACC is a hydrogen bond acceptor). Subsequent columns specify the position of the feature center in a three-dimensional space. Individual pharmacophores are separated by lines containing four dollar signs (*$$$$*).
```

Comment:
with reference to my comment in the introduction - maybe you can expand here?

Reply:
In the tutorial, after uploading the ePharmaLib dataset, I suggested to the users to view its contents by pressing the eye icon (View data).